.cloudgov/manifest.yml

```diff
@@ -34,14 +33,14 @@ applications:
     env:
       NODE_ENV: production
       APP_ENV: ((env))
-      APP_HOSTNAME: https://federalistapp((env_postfix)).18f.gov
+      APP_HOSTNAME: https://((app_subdomain))((hostname))
```

Comment:
nit: I think `hostname` is actually a domain, so maybe it should be called `domain` or something along those lines?

Reply:
+1 changed to `domain`